### ISSUE TITLE
New feature: Restart command

### ISSUE THREAD

**Issue by thomasplevy** [louisgab/devilbox-cli#5]
I wrote a simple bash command to restart devilbox via the cli, `devilbox stop && devilbox run -s`. In the last week I've found myself regularly running this command.

If it seems generally useful I added a new command to the devilbox-cli to enable this in a single command:

```sh
devilbox restart    # Stop and restart containers.
devilbox restart -s # Stop and restart containers silently.
```

I've already committed the command to my fork and if it feels useful I'll submit it as a PR instead of working off my fork locally.

Here's the diff if you're interested: https://github.com/louisgab/devilbox-cli/compare/master...gocodebox:feature/restart-command?expand=1

However I accidentally committed after submitting #4 and I don't feel like rewriting history on my fork to get a clean diff without the changes made to #4 

Thanks! This is a great tool and I just want to make it a little better for my personal needs

**Comment by louisgab** [comment]
I just merged your awesome command in #6 and released #4 and #6 in new release 0.3.0, now available on npm
Thanks again for your help on the project

**Comment by thomasplevy** [comment]
@louisgab beautiful! Thanks so much!
